chia/cmds/plotnft_funcs.py

```diff
@@ -270,106 +274,133 @@ async def submit_tx_with_confirmation(
+        selected_wallet_id,
+        bytes32.from_hexstr(json_dict["target_puzzle_hash"]),
+        pool_url,
+        json_dict["relative_lock_height"],
+        fee,
+    )
 
-async def claim_cmd(*, wallet_rpc_port: Optional[int], fingerprint: int, fee: uint64, wallet_id: int) -> None:
-    async with get_wallet_client(wallet_rpc_port, fingerprint) as (wallet_client, fingerprint, _):
-        msg = f"\nWill claim rewards for wallet ID: {wallet_id}."
-        func = functools.partial(
-            wallet_client.pw_absorb_rewards,
-            wallet_id,
-            fee,
+    await submit_tx_with_confirmation(
+        msg, prompt, func, wallet_info.client, wallet_info.fingerprint, selected_wallet_id
+    )
+
+
+async def self_pool(*, wallet_info: WalletClientInfo, fee: uint64, wallet_id: Optional[int], prompt: bool) -> None:
+    selected_wallet_id = await wallet_id_lookup_and_check(wallet_info.client, wallet_id)
+    msg = (
+        "Will start self-farming with Plot NFT on wallet id "
+        f"{selected_wallet_id} fingerprint {wallet_info.fingerprint}."
+    )
+    func = functools.partial(wallet_info.client.pw_self_pool, selected_wallet_id, fee)
+    await submit_tx_with_confirmation(
+        msg, prompt, func, wallet_info.client, wallet_info.fingerprint, selected_wallet_id
+    )
+
+
+async def inspect_cmd(wallet_info: WalletClientInfo, wallet_id: Optional[int]) -> None:
+    selected_wallet_id = await wallet_id_lookup_and_check(wallet_info.client, wallet_id)
+    pool_wallet_info, unconfirmed_transactions = await wallet_info.client.pw_status(selected_wallet_id)
+    print(
+        json.dumps(
+            {
+                "pool_wallet_info": pool_wallet_info.to_json_dict(),
+                "unconfirmed_transactions": [
+                    {"sent_to": tx.sent_to, "transaction_id": tx.name.hex()} for tx in unconfirmed_transactions
+                ],
+            }
         )
-        await submit_tx_with_confirmation(msg, False, func, wallet_client, fingerprint, wallet_id)
+    )
+
+
+async def claim_cmd(*, wallet_info: WalletClientInfo, fee: uint64, wallet_id: Optional[int]) -> None:
+    selected_wallet_id = await wallet_id_lookup_and_check(wallet_info.client, wallet_id)
+    msg = f"\nWill claim rewards for wallet ID: {selected_wallet_id}."
+    func = functools.partial(
+        wallet_info.client.pw_absorb_rewards,
+        selected_wallet_id,
+        fee,
+    )
+    await submit_tx_with_confirmation(msg, False, func, wallet_info.client, wallet_info.fingerprint, selected_wallet_id)
 
 
-async def change_payout_instructions(launcher_id: str, address: CliAddress) -> None:
+async def change_payout_instructions(launcher_id: bytes32, address: CliAddress, root_path: Optional[Path]) -> None:
     new_pool_configs: list[PoolWalletConfig] = []
     id_found = False
     puzzle_hash = address.validate_address_type_get_ph(AddressType.XCH)
+    if root_path is None:
+        root_path = DEFAULT_ROOT_PATH  # pragma: no cover
```

Comment:
I don't think this is a thing that should inherently never be verified to work.

```suggestion
        root_path = DEFAULT_ROOT_PATH
```

Or maybe the optionality should be handled outside of here and the path is just non-optional?  Feels like one way or another we shouldn't need to reimplement this everywhere.

Reply:
I just don't know how to test this in CI in order to get this covered. No matter where this is, CI will never trigger the DEFAULT_ROOT_PATH line since root path is always explicitly set (afaik)

Reply:
I'm not too worried about whether or not you test it.  But, since it isn't going to be tested it seems worth reminding folks in any future PRs that modify it etc.  And to have it shown as not tested in coverage reports.  Also, I'm pretty sure I could make a test that would test it, but sure, it probably wouldn't fit existing patterns and such.  Might have to call the function directly.

But mostly I reserve the no cover pragma for places that shouldn't be covered.  Like code that should only execute if there is a bug and it's purpose is to notify folks of that bug.

Reply:
Removed and added a mock to test it